requirements.txt

```diff
@@ -0,0 +1,18 @@
+asn1crypto==0.24.0
+Click==7.0
+cryptography==2.1.4
+enum34==1.1.6
+Flask==1.0.2
+idna==2.6
+ipaddress==1.0.17
+itsdangerous==1.1.0
+Jinja2==2.10.1
+keyring==10.6.0
+keyrings.alt==3.0
+MarkupSafe==1.1.1
+pycrypto==2.6.1
+pygobject==3.26.1
+pyxdg==0.25
+SecretStorage==2.3.1
+six==1.11.0
+Werkzeug==0.15.2
```

Comment:
this is correct, but very verbose (i.e. it lists every resolved dependency of flask). we can just require flask and allow it to figure out the flask dependencies itself.